src/platforms/ctxlink/platform.c

```diff
@@ -191,6 +248,25 @@ void platform_init(void)
 
 	platform_timing_init();
 	adc_init();
+	wifi_init(); //	Setup the Wifi channel
+#ifdef WINC_1500_FIRMWARE_UPDATE
+	//
+	// ONLY for firmware update
+	//
+	// Perform WINC1500 reset sequence
+	//
+	m2mStub_PinSet_CE(M2M_WIFI_PIN_LOW);
+	m2mStub_PinSet_RESET(M2M_WIFI_PIN_LOW);
+	DelayMs(100);
```

Comment:
The delay routine is the one used in the WINC1500 driver source and this small piece of code was taken from their app note explaining how to update the WINC1500 firmware. CHanged to use the platform_delay, since the file it is in is outside of the WINC1500 driver.

Reply:
Ahhh, okay, that makes sense of that. Thank you!